src/pudl/transform/eia860.py

```diff
@@ -1157,3 +1157,172 @@ def cooling_equipment_continuity(cooling_equipment):
+    # earlier years.
+    for sorbent_col in (col for col in fgd_df.columns if "sorbent_type" in col):
+        fgd_df.loc[fgd_df[sorbent_col] == "WT", sorbent_col] = "W"
+
+    # Handle some non-numeric values in the pond landfill requirements column
+    fgd_df["pond_landfill_requirements_acre_foot_per_year"] = pd.to_numeric(
+        fgd_df.pond_landfill_requirements_acre_foot_per_year, errors="coerce"
+    )
+
+    fgd_df = (
+        pudl.metadata.classes.Package.from_resource_ids()
+        .get_resource("_core_eia860__fgd_equipment")
+        .encode(fgd_df)
+    )
+
+    return fgd_df.pipe(apply_pudl_dtypes, strict=True)
+
+
+@asset_check(asset=_core_eia860__fgd_equipment, blocking=True)
+def fgd_equipment_null_check(fgd):
+    """Check that columns other than expected columns aren't null."""
+    fast_run_null_cols = {
+        "county",
+        "fgd_operational_status_code",
+        "operating_date",
+        "fgd_manufacturer_code",
+        "plant_summer_capacity_mw",
+        "water_source",
+    }
+    if fgd.report_date.min() >= pd.Timestamp("2011-01-01T00:00:00"):
+        expected_cols = set(fgd.columns) - fast_run_null_cols
+    else:
+        expected_cols = set(fgd.columns)
+    pudl.validate.no_null_cols(fgd, cols=expected_cols)
+    return AssetCheckResult(passed=True)
+
+
+@asset_check(asset=_core_eia860__fgd_equipment, blocking=True)
+def fgd_cost_discrepancy_check(fgd):
+    """Costs should sum to cost_total.
+
+    To allow for *some* data quality errors we assert that costs ~=
+    total cost at least 99% of the time (with a 1% acceptable
+    discrepancy).
+    """
+
+    def sum_to_target_rate(sum_cols, target, threshold):
+        discrepancies = (
+            fgd.loc[:, sum_cols].sum(axis="columns") - fgd.loc[:, target]
+        ).dropna() / fgd.loc[:, target]
+        return (discrepancies > threshold).sum() / len(discrepancies)
+
+    opex_cost_discrepancy_rate = sum_to_target_rate(
+        sum_cols=[col for col in fgd if "cost_" in col and "total" not in col],
+        target="total_fgd_equipment_cost",
+        threshold=0.01,
+    )
+    logger.info(f"Observed FGD cost discrepancy: {opex_cost_discrepancy_rate}")
+    assert opex_cost_discrepancy_rate < 0.01
+
+    return AssetCheckResult(passed=True)
+
+
+@asset_check(asset=_core_eia860__fgd_equipment, blocking=True)
+def fgd_equipment_continuity(fgd):
+    """Check to see if columns vary as slowly as expected."""
+    return pudl.validate.group_mean_continuity_check(
+        df=fgd,
+        thresholds={
+            "flue_gas_exit_rate_cubic_feet_per_minute": 0.1,
+            "flue_gas_exit_temperature_fahrenheit": 0.1,
+            "pond_landfill_requirements_acre_foot_per_year": 0.1,
+            "so2_removal_efficiency_design": 0.1,
+            "so2_emission_rate_lbs_per_hour": 0.1,
+            "specifications_of_coal_ash": 0.1,
+            "specifications_of_coal_sulfur": 7,  # To do: investigate
```

Comment:
nit: somehow by convention `TODO` is how you mark todo's in code, and various editors/tooling all look for that specific word. so maybe worth doing this:

```suggestion
            "specifications_of_coal_sulfur": 7,  # TODO (2024-03-14): investigate
```